apps/api/package.json

```diff
@@ -11,7 +11,7 @@
     "dev": "vite",
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
-    "console": "yarn start -- --watch --entryFile repl",
```

Comment:
I feel like I'm swapping between yarn and npm, but I guess we're using npm now? 😃

Reply:
hahaha sorry I should have left a note here.. yarn was failing to capture the terminal session when using the REPL. For some reason, running this command specifically with npm works and I didn't want to put effort to investigate why and fix it.

We're definitely using yarn and you should still run `yarn console`

Reply:
Let's keep a proper ticket here :D https://github.com/mathiasberggren/streaming-subscription/issues/23